### ISSUE TITLE
Additional metadata info to add to user experience guide

### ISSUE THREAD

**Issue by avneeshsingh** [w3c/publ-a11y#137]
One of the recommendation from the user groups was to display additional metadata information which is not part of accessibility metadata, but it helps the users with accessibility needs. These are as follows:
* File format (PDF, Kindle, EPUB 2, EPUB 3 etc.)
* Refloable or fixed layout
* Availability of pagelist and headings
* Tables with text instead of tables in form of images.
* Use of DRM and which kind of protection is used because some DRM/protection can block accessibility.

We should finalize this list, and then figure out how it will be arranged in level 1 and level 2 #136 .

**Comment by gautierchomel** [comment]
_Proposal in PR #210 copy pasted here for easier review by everyone._

### General information (supplement the bibliographic information)

The product details are giving precious information about usability of the book in relation with specifics needs. The following informations should always be displayed:

- File format (EPUB, PDF, MP3, Audiobook, etc.)
- Protection measure or lack thereof
- Date of publication
- Name of the publishing house
- Main language of the content

### Why this information is important for accessibility? _(note: in the code this part is a Detail, this title is the summary)_

- The file format gives a strong indication of accessibility: an MP3 format audio book will be less structured than an Audiobook; a PDF does not allows for fine typography tunning, etc.
- The protection measure may block assistive technologies such as screen readers. In addition, many specific reading devices such as DAISY readers or Braille notepads are not equipped to read encrypted files.
- The publication date gives an indication of when the files were produced. A book published before 2011 will probably be an EPUB2 file including poor semantics, and a book published in 2022 will be more likely to be correctly structured and benefit of strong semantics.
- The name of the publishing house can highlight the efforts it has made in terms of accessibility.
- The main language of the content enables readers to be sure that they will be able to read in good conditions by checking that they have a synthesised voice or the corresponding Braille table.

**Comment by GeorgeKerscher** [comment]
We address the items here in the new Accessibility Metadata Display Guide for Digital Publications 2.0
